src/test/app/RCLValidations_test.cpp

```diff
@@ -35,12 +35,19 @@ class RCLValidations_test : public beast::unit_test::suite
     testChangeTrusted()
     {
         testcase("Change validation trusted status");
-        PublicKey key = derivePublicKey(KeyType::ed25519, randomSecretKey());
-        auto v = std::make_shared<STValidation>(uint256(), uint256(),
-            NetClock::time_point(), key, calcNodeID(key), true);
+        auto keys = randomKeyPair(KeyType::secp256k1);
```

Comment:
Was the key type change intentional?

Reply:
Yes, the old test never actually signed anything with the key. Currently, only `secp256k1` keys can be used to sign validation messages.